test/utils.test.js

```diff
@@ -17,7 +16,7 @@ test('should take in a drink and return a li with relevant information', (expect
         price: 8,        
     };
 
-    const expected = '<li id="pinaColada" class="drink"><h3 class="name">Piña colada</h3><img class="image" src="../assets/pinaColada.jpg"><p class="description">A blended pineapple and coconut classic, originated in Puerto Rico.</p><p class="price">$8.00</p><button>Add</button></li>';
+    const expected = '<li id="pinaColada" class="drink"><h3 class="name">Piña colada</h3><img class="image" src="../assets/pinaColada.jpg"><p class="description">A blended pineapple and coconut classic, originated in Puerto Rico.</p><p class="price">$8.00</p><button>Add</button><select class="select"><option value="1">1</option><option value="2">2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option></select></li>';
```

Comment:
looks like this test still is not passing right now.  I think it will be a small fix :)